src/modules/clusters/cluster-list/cluster-list.js

```diff
@@ -364,6 +361,25 @@
                 cluster.isProfilingEnabled === "Unknown";
         }
 
+        function getTemplate(cluster) {
+            var tooltip = {
+                    "unmanagecluster": "If unmanage fails, resolve the issue and reinitiate unmanage cluster.",
+                    "importcluster": "If import cluster fails, resolve the issue before performing an unmanage cluster and reinitiate import.",
+                    "expandclusterwithdetectedpeers": "If cluster expansion fails, check if tendrl-ansible was executed successfully and ensure the node agents are correctly configured.",
+                    "ready": "The cluster is successfully imported for viewing monitoring data and metrics."
+                },
+                template = "";
+
+            if (cluster.currentStatus === "failed") {
+                template = tooltip[cluster.jobType.toLowerCase()];
+
+            } else if (cluster.managed === "Yes" && cluster.currentStatus !== "failed" && cluster.currentStatus !== "in_progress") {
+                template = tooltip["ready"];
+            }
+
```

Comment:
@cloudbehl Because `finished` can be for various jobs like `ImportCluster`, 'UnmanageCluster`, `ProfilingTask` etc. Its not necessary that if the currentTask is finished, the cluster is ready to use. Hence, to make  sure, this condition is needed. 
Again, I think string comparison with "Ready to Use" will be better here as even if the profiling task is in progress or failed, the cluster will be still in "Ready to use" state and hence the tooltip needs to be shown. Just checking `managed==='Yes'` will not solve it as for 'expandclusterwithdetectedpeers' job's in_progress state, the message displayed is "Expansion in progress" and not "Ready to use"